### ISSUE TITLE
User story: using available identifiers

### ISSUE THREAD

**Issue by drkane** [ThreeSixtyGiving/ID-reconcile#7]
*As a* user of 360 giving data
*I want to* know that every organisation with a possible external identifier uses it
*So that* I can be confident that organisations without an external identifier aren't on external registers

**Comment by morchickit** [comment]
Question - why are we using the term external identifiers here like in user story #2?

**Comment by morchickit** [comment]
Also, we need to be more specific with the word user - analyst? A coder?

**Comment by drkane** [comment]
> Question - why are we using the term external identifiers here like in user story #2?

I'm drawing a distinction between valid "internal" identifiers (like `360G-DaveFoundation-12345`) and identifiers that come from external sources. I think in user story #2 it's more about making sure they have any identifier.

**Comment by morchickit** [comment]
DQT does it to an identifier, but not external identifiers at the moment, no? It can't compare against a list of id and make sure the one that was given is valid, no?

**Comment by robredpath** [comment]
@morchickit that's correct - DQT will tell you if you're using 360G- prefixed IDs, but won't check your external IDs against any external services.
